.github/workflows/build-binaries-and-brew.yaml

```
@@ -8,10 +8,24 @@ jobs:
-        # We build on an older Ubuntu as pyinstaller binaries are forward-compatible not backwards-compatible
-        # See https://pyinstaller.org/en/stable/usage.html?highlight=glibc#making-gnu-linux-apps-forward-compatible:~:text=The%20solution%20is%20to%20always%20build%20your%20app%20on%20the%20oldest%20version%20of%20GNU/Linux%20you%20mean%20to%20support.%20It%20should%20continue%20to%20work%20with%20the%20libc%20found%20on%20newer%20versions.
-        # TODO: for similar reasons, we may want to build on older Windows/MacOS versions as well
-        os: [ubuntu-20.04, windows-latest, macos-latest]
+        include:
+          # Linux AMD64 - build on older Ubuntu for forward compatibility
+          # See https://pyinstaller.org/en/stable/usage.html?highlight=glibc#making-gnu-linux-apps-forward-compatible
+          - os: ubuntu-22.04
+            platform: linux
+            arch: amd64
+          # macOS AMD64 (Intel) - macos-13 is the last Intel-based runner
+          - os: macos-13
```

actionlint (1.7.10)

[error] label "macos-13" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-xlarge", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)